worlds/messenger/options.py

```diff
@@ -30,17 +31,23 @@ class PortalPlando(PlandoConnections):
     portals = [f"{portal} Portal" for portal in PORTALS]
     shop_points = [point for points in SHOP_POINTS.values() for point in points]
     checkpoints = [point for points in CHECKPOINTS.values() for point in points]
-    portal_entrances = PORTALS
-    portal_exits = portals + shop_points + checkpoints
-    entrances = portal_entrances
-    exits = portal_exits
 
+    entrances = PORTALS
+    exits = portals + shop_points + checkpoints
 
-# for back compatibility. To later be replaced with transition plando
-class HiddenPortalPlando(PortalPlando):
-    visibility = Visibility.none
-    entrances = PortalPlando.entrances
-    exits = PortalPlando.exits
+
+class TransitionPlando(PlandoConnections):
+    """
+    Plando connections to be used with transition shuffle.
+    List of valid connections can be found at https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/messenger/connections.py#L641.
```

Comment:
Links in docstrings do not work whatsoever on webhost. I'd recommend changing the game page

Reply:
I'd still like to leave them here because they show up in the yaml as well, but I'm also planning to add a plando guide (preferably as a separate PR) since I have so many plando options now.

Reply:
Making me really wish we could have separate docstrings for both XD
Fair enough